conda/base/constants.py

```diff
@@ -42,7 +43,9 @@
     '$CONDA_PREFIX/condarc',
     '$CONDA_PREFIX/condarc.d/',
     '$CONDARC',
-)
+]
+if on_win:
+    SEARCH_PATH += ['C:/ProgramData/conda/.condarc']
```

Comment:
Probably best to keep it immutable and list the paths explicitly got each platform. Follow the pattern set by `DEFAULT_CHANNELS` below.

Reply:
I guess `COMPATIBLE_SHELLS` uses a different pattern. Would still prefer to keep the values immutable.